packages/cli/src/index.ts

```diff
@@ -29,7 +29,15 @@ yargs(process.argv.slice(2))
   })
   .command({
     command: 'push',
-    builder: () => yargs.options({ branch: branchDefinition }),
+    builder: () =>
+      yargs.options({
+        branch: branchDefinition,
+        'delete-unused-keys': {
+          type: 'boolean',
+          describe: 'Whether or not to delete unused keys after pushing',
```

Comment:
Does the CLI already tell you what the default is? If not I like adding it to the description as it's usually an important thing to know.

```suggestion
          describe: 'Whether or not to delete unused keys after pushing. Default: false',
```

Reply:
Yargs handles this for you.
![image](https://user-images.githubusercontent.com/5663042/190308477-99b6b043-001a-47a4-960a-1899e079d7f5.png)